### DISCUSSION TITLE
converting standard sql query to livewire query

### DISCUSSION THREAD

**Discussion by alpaygurogluods** [livewire/livewire#5185] [3. Feature ideas]
We cannot translate our code below into a livewire query. Because we couldn't find any documentation specifying the properties that the sql query commands take. Our query is as follows:

`$datalist = DB::select('Select meeting_notes.id,
               meeting_notes.id as id,
               meeting_notes.creater_id,            
               creater.name as createrName,
               meeting_notes.date,
               meeting_notes.startHour,            
               meeting_notes.endHour,
               meeting_notes.noteCurrentProject,            
               meeting_notes.anyRequest,
               meeting_notes.requestDescription,
               meeting_notes.requestSolution,
               meeting_notes.requestOrProblem,            
               meeting_notes.type,            
               meeting_notes.visitingType,            
               meeting_notes.doesCrossSelling,
               meeting_notes.opportunityTransfer,
               meeting_notes.project_id,
               projects.id as project_id          
                     from meeting_notes,projects,creater    
                     WHERE            
                        (meeting_notes.creater_id = creater.id AND
                        meeting_notes.project_id = projects.id) AND
                       (meeting_notes.project_id = ' . $id . ') AND (meeting_notes.module = "PM")
                        group by meeting_notes.id                
                ');
`

**Comment by ju5t** [comment]
What do you mean by a 'Livewire query'? There is no such thing. You can use Eloquent and models for it. See https://laravel.com/docs/9.x/eloquent.

**Comment by joshhanley** [comment]
@alpaygurogluods if you are trying to save data from your above query into a Livewire property, then yeah that won't work.

Livewire only supports storing basic Eloquent collections, which get returned from a query called on an Eloquent model, such as  `User::where(...)->get()`.

I would suggest adding your query to a computed property instead and using that. https://laravel-livewire.com/docs/2.x/properties#computed-properties

Hope this helps!
